src/core/Directus/Database/Schema/Object/FieldRelationship.php

```diff
@@ -109,6 +120,7 @@ public function isOneToMany()
     protected function guessType()
     {
         $fieldName = $this->fromField->getName();
+        $interface = $this->fromField->attributes->get('interface');
```

Comment:
This is a good idea, but this will only be attached to an interface, and not the `m2m` type (which doesn't exists). In this version we are trying to avoid attaching logic based on interface names in the core unless it's an extension.